### ISSUE TITLE
Export to other file formats and network expansion of QGIS users

### ISSUE THREAD

**Issue by IltonMarcos** [qgis/QGIS#50306]
### Feature description

Like QGIS that is known and loved in the GIS world, Blender, WebODM and CloudCompare are powerful, free software with vibrant communities around the world. I believe that exporting products to formats that support these software would also be a way to attract more users to QGIS.
Exporting products (especially texts) from QGIS to these software would shorten the distance between these communities.
I suggest native export to formats such as: .blend, .obj, .ply, .stl, .fbx, .gltf and .bin(CloudCompare) as is already done for .svg and .pdf.

Another idea would be to export map texts in Print Composer to HTML or Rich Text.

**Comment by baswein** [comment]
I agree with the idea of this feature request and am often trying to get things back and forth between blender and QGIS but this I think this is really not focused enough to be useful to keep open. I think each of these formats should be evaluated on thier merits and justified with a feature request. 

Also I wanted to give a quick update on the status of some of these formats from what I have seen in QGIS: 
1. Conversion to .ply is now possible through the TIN Mesh Creation algorithm. 
2. Export to .obj is possible through export of a 3D map. And maybe .stl in the future - https://github.com/qgis/QGIS/pull/63444 
3. gltf is available with the qgis2threejs plugin. 

Maybe those are enough for now. These are file formats that are in the underlying libraries and so are a relatively easy lift I would be surprised if .blend and .bin were and so are unlikely to be implemented. I wonder if the recently added SFCGAL library will expose more export format options.
